spec/fixtures/searchgov_urls.yml

```diff
@@ -16,3 +16,9 @@ enqueued:
   last_crawled_at: <%= 1.day.ago.to_s(:db) %>
   lastmod: <%= 1.week.ago.to_s(:db) %>
   enqueued_for_reindex: true
+
+crawled_more_than_month:
+  url: http://www.agency.gov/crawled_more_than_month
+  last_crawled_at: <%= 30.days.ago.to_s(:db) %>
+  lastmod: <%= 30.day.ago.to_s(:db) %>
```

Comment:
Let's skip setting a `lastmod` value in this fixture. It's not necessary for what we need to test, and it unnecessarily complicates the spec setup. For example, if you change `last_crawled_at` to `45.days.ago`, you'll need to also change `lastmod` to a value greater than 45 days ago to ensure that your spec isn't a false positive (because a URL with a `lastmod` value more recent than its `last_crawled_at` would automatically be included in the `fetch_required` scope).

Reply:
Yeah I set that last mod to that date to prevent false positives but I see your point in just taking the whole thing out.